mods/common/chrome/settings-display.yaml

```diff
@@ -1,224 +1,317 @@
+				Container@ROW:
+					Width: PARENT_RIGHT - 24
+					Height: 50
+					Children:
+						Container@BATTLEFIELD_CAMERA_DROPDOWN_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Label@BATTLEFIELD_CAMERA:
+									Width: PARENT_RIGHT
+									Height: 20
+									Text: Battlefield Camera:
+								DropDownButton@BATTLEFIELD_CAMERA_DROPDOWN:
+									Y: 25
+									Width: PARENT_RIGHT
+									Height: 25
+									Font: Regular
+						Container@TARGET_LINES_DROPDOWN_CONTAINER:
+							X: PARENT_RIGHT / 2 + 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Label@TARGET_LINES:
+									Width: PARENT_RIGHT
+									Height: 20
+									Text: Target Lines:
+								DropDownButton@TARGET_LINES_DROPDOWN:
+									Y: 25
+									Width: PARENT_RIGHT
+									Height: 25
+									Font: Regular
+				Container@ROW:
+					Width: PARENT_RIGHT - 24
+					Height: 50
+					Children:
+						Container@UI_SCALE_DROPDOWN_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Label@UI_SCALE:
+									Width: PARENT_RIGHT
+									Height: 20
+									Text: UI Scale:
+								DropDownButton@UI_SCALE_DROPDOWN:
+									Y: 25
+									Width: PARENT_RIGHT
+									Height: 25
+									Font: Regular
+						Container@STATUS_BAR_DROPDOWN_CONTAINER:
+							X: PARENT_RIGHT / 2 + 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Label@STATUS_BARS:
+									Width: PARENT_RIGHT
+									Height: 20
+									Text: Status Bars:
+								DropDownButton@STATUS_BAR_DROPDOWN:
+									Y: 25
+									Width: PARENT_RIGHT
+									Height: 25
+									Font: Regular
+				Container@ROW:
+					Width: PARENT_RIGHT - 24
+					Height: 20
+					Children:
+						Container@CURSORDOUBLE_CHECKBOX_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Checkbox@CURSORDOUBLE_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Increase Cursor Size
+						Container@PLAYER_STANCE_COLORS_CHECKBOX_CONTAINER:
+							X: PARENT_RIGHT / 2 + 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Checkbox@PLAYER_STANCE_COLORS_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Player Stance Colors
+				Container@ROW:
+					Width: PARENT_RIGHT - 24
+					Height: 20
+					Children:
+						Container@PAUSE_SHELLMAP_CHECKBOX_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 10
+							Children:
+								Checkbox@PAUSE_SHELLMAP_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Pause Menu Background
+						Container@UI_FEEDBACK_CHECKBOX_CONTAINER:
+							X: PARENT_RIGHT / 2 + 10
+							Width: PARENT_RIGHT / 2 - 20
+							Children:
+								Checkbox@UI_FEEDBACK_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: UI Feedback in Transients Panel
```

Comment:
@pchote not entirely sure, but shouldn't this be `Transient Panels`?

Reply:
I think Transients Panel is indeed correct, since this is referring to the new ingame display for transient feedback.

Reply:
It does sound somewhat unclear. Should we change to something like "Show UI Feedback Notifications"? Or add a tooltip description? (not necessarily in this PR)

Reply:
> Show UI Feedback Notifications

👍

Reply:
Sounds good, but I'll merge this first then (there are some other improvements I'd like to see as well, I'll file a follow-up PR or issue about it later).